k-distribution/tests/regression-new/map-symbolic-tests-haskell/lookup-9-spec.k.out

```diff
@@ -1 +1,47 @@
-#Top
+  #Not ( {
+    X
+  #Equals
+    Y
+  } )
+#And
+  #Not ( {
+    X
+  #Equals
+    Z
+  } )
+#And
+  #Not ( {
+    Y
+  #Equals
+    Z
+  } )
+#And
+  <k>
+    lookupResult ( MAP [ Z:MyId ] ~> . ) ~> .
+  </k>
+#And
+  {
+    false
+  #Equals
+    X:MyId in_keys ( MAP )
+  }
+#And
+  {
+    true
+  #Equals
+    Z:MyId in_keys ( ( MAP
+    X:MyId |-> 1 ) [ Y:MyId <- 2 ] )
+  }
+#And
+  {
+    true
+  #Equals
+    Z:MyId in_keys ( MAP )
+  }
+#And
+  {
+    true
+  #Equals
+    Z:MyId in_keys ( MAP [ Y:MyId <- 2 ]
+    ( X:MyId |-> 1 ) )
+  }
```

Comment:
This is a little weird. Strictly, we only need the `{ true #Equals Z:MyId in_keys ( MAP ) }` side condition.